pkg/windows/windows.go

```diff
@@ -177,6 +179,7 @@ func getFilesToTransfer() (map[*payload.FileInfo]string, error) {
 		payload.KubeProxyPath:                  K8sDir,
 		payload.KubeletPath:                    K8sDir,
 		payload.AzureCloudNodeManagerPath:      K8sDir,
+		payload.KubeLogRunnerPath:              K8sDir,
```

Comment:
Should we also add this to [requiredFiles](https://github.com/openshift/windows-machine-config-operator/blob/master/cmd/operator/main.go#L111)?
Although I'm seeing that var is quite outdated so maybe it warrants a task of its own to update properly.